sematic/ui/packages/common/src/component/tests/test_RunsDropdown.cy.tsx

```diff
@@ -2,7 +2,7 @@ import RunsDropdown from "../RunsDropdown";
 
 describe("RunStateChip component", () => {
     it("should render", () => {
-        cy.mount(<RunsDropdown />);
+        cy.mount(<RunsDropdown runs={[]} />);
```

Comment:
Do you have a way of keeping track of all the code locations that need to be updated in the future?